2024-08-07 18:12:12,733 INFO    StreamThr :19868 [internal.py:wandb_internal():86] W&B internal server running at pid: 19868, started at: 2024-08-07 18:12:12.733229
2024-08-07 18:12:12,734 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: status
2024-08-07 18:12:12,734 INFO    WriterThread:19868 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/run-ag4ny1nq.wandb
2024-08-07 18:12:12,735 DEBUG   SenderThread:19868 [sender.py:send():382] send: header
2024-08-07 18:12:12,770 DEBUG   SenderThread:19868 [sender.py:send():382] send: run
2024-08-07 18:12:12,954 INFO    SenderThread:19868 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files
2024-08-07 18:12:12,954 INFO    SenderThread:19868 [sender.py:_start_run_threads():1136] run started: ag4ny1nq with start time 1723025532.733121
2024-08-07 18:12:12,966 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: check_version
2024-08-07 18:12:12,966 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: check_version
2024-08-07 18:12:13,066 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: run_start
2024-08-07 18:12:13,079 DEBUG   HandlerThread:19868 [system_info.py:__init__():32] System info init
2024-08-07 18:12:13,079 DEBUG   HandlerThread:19868 [system_info.py:__init__():47] System info init done
2024-08-07 18:12:13,079 INFO    HandlerThread:19868 [system_monitor.py:start():194] Starting system monitor
2024-08-07 18:12:13,079 INFO    SystemMonitor:19868 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-07 18:12:13,079 INFO    HandlerThread:19868 [system_monitor.py:probe():214] Collecting system info
2024-08-07 18:12:13,080 INFO    SystemMonitor:19868 [interfaces.py:start():190] Started cpu monitoring
2024-08-07 18:12:13,080 INFO    SystemMonitor:19868 [interfaces.py:start():190] Started disk monitoring
2024-08-07 18:12:13,080 INFO    SystemMonitor:19868 [interfaces.py:start():190] Started gpu monitoring
2024-08-07 18:12:13,080 INFO    SystemMonitor:19868 [interfaces.py:start():190] Started memory monitoring
2024-08-07 18:12:13,081 INFO    SystemMonitor:19868 [interfaces.py:start():190] Started network monitoring
2024-08-07 18:12:13,089 DEBUG   HandlerThread:19868 [system_info.py:probe():196] Probing system
2024-08-07 18:12:13,091 DEBUG   HandlerThread:19868 [system_info.py:_probe_git():181] Probing git
2024-08-07 18:12:13,095 DEBUG   HandlerThread:19868 [system_info.py:_probe_git():189] Probing git done
2024-08-07 18:12:13,095 DEBUG   HandlerThread:19868 [system_info.py:probe():244] Probing system done
2024-08-07 18:12:13,095 DEBUG   HandlerThread:19868 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-07T10:12:13.090041', 'startedAt': '2024-08-07T10:12:12.727679', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/llama2-7b', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '4000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1387.4977857142856, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 4878.53, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5215.385, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 3891.677, 'min': 800.0, 'max': 5500.0}, {'current': 3184.483, 'min': 800.0, 'max': 5500.0}, {'current': 3118.238, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5413.278, 'min': 800.0, 'max': 5600.0}, {'current': 4304.187, 'min': 800.0, 'max': 5600.0}, {'current': 1461.07, 'min': 800.0, 'max': 5600.0}, {'current': 1255.391, 'min': 800.0, 'max': 5600.0}, {'current': 1148.626, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1135.01, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1191.565, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 474.66730880737305}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-07 18:12:13,096 INFO    HandlerThread:19868 [system_monitor.py:probe():224] Finished collecting system info
2024-08-07 18:12:13,096 INFO    HandlerThread:19868 [system_monitor.py:probe():227] Publishing system info
2024-08-07 18:12:13,096 DEBUG   HandlerThread:19868 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-07 18:12:13,096 DEBUG   HandlerThread:19868 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-07 18:12:13,096 DEBUG   HandlerThread:19868 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-07 18:12:13,954 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/requirements.txt
2024-08-07 18:12:13,955 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/conda-environment.yaml
2024-08-07 18:12:14,779 DEBUG   HandlerThread:19868 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-07 18:12:14,779 DEBUG   HandlerThread:19868 [system_info.py:_save_code():90] Saving code
2024-08-07 18:12:14,780 DEBUG   HandlerThread:19868 [system_info.py:_save_code():111] Saving code done
2024-08-07 18:12:14,780 DEBUG   HandlerThread:19868 [system_info.py:_save_patches():128] Saving git patches
2024-08-07 18:12:14,809 DEBUG   HandlerThread:19868 [system_info.py:_save_patches():170] Saving git patches done
2024-08-07 18:12:14,809 INFO    HandlerThread:19868 [system_monitor.py:probe():229] Finished publishing system info
2024-08-07 18:12:14,810 DEBUG   SenderThread:19868 [sender.py:send():382] send: files
2024-08-07 18:12:14,810 INFO    SenderThread:19868 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-07 18:12:14,810 INFO    SenderThread:19868 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-07 18:12:14,811 INFO    SenderThread:19868 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-07 18:12:14,814 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: stop_status
2024-08-07 18:12:14,814 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-07 18:12:14,815 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: stop_status
2024-08-07 18:12:14,954 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/conda-environment.yaml
2024-08-07 18:12:14,954 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/code/llama/gmm.py
2024-08-07 18:12:14,954 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/wandb-metadata.json
2024-08-07 18:12:14,954 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/diff.patch
2024-08-07 18:12:14,954 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/code/llama
2024-08-07 18:12:14,955 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/code
2024-08-07 18:12:14,988 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62
2024-08-07 18:12:14,988 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5216', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:14,988 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96A98570DAAD7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: telemetry
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: config
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: metric
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: telemetry
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: metric
2024-08-07 18:12:15,006 WARNING SenderThread:19868 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: telemetry
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: telemetry
2024-08-07 18:12:15,006 DEBUG   SenderThread:19868 [sender.py:send():382] send: config
2024-08-07 18:12:15,153 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025
2024-08-07 18:12:15,153 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:15,153 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96A985D0F27D7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:15,172 DEBUG   SenderThread:19868 [sender.py:send():382] send: exit
2024-08-07 18:12:15,172 INFO    SenderThread:19868 [sender.py:send_exit():589] handling exit code: 1
2024-08-07 18:12:15,172 INFO    SenderThread:19868 [sender.py:send_exit():591] handling runtime: 2
2024-08-07 18:12:15,173 INFO    SenderThread:19868 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:12:15,173 INFO    SenderThread:19868 [sender.py:send_exit():597] send defer
2024-08-07 18:12:15,173 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,173 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-07 18:12:15,173 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,173 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-07 18:12:15,173 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 1
2024-08-07 18:12:15,173 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,173 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-07 18:12:15,173 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,173 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-07 18:12:15,173 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 2
2024-08-07 18:12:15,173 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,173 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-07 18:12:15,173 INFO    HandlerThread:19868 [system_monitor.py:finish():203] Stopping system monitor
2024-08-07 18:12:15,173 DEBUG   SystemMonitor:19868 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-07 18:12:15,173 INFO    HandlerThread:19868 [interfaces.py:finish():202] Joined cpu monitor
2024-08-07 18:12:15,173 DEBUG   SystemMonitor:19868 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-07 18:12:15,174 INFO    HandlerThread:19868 [interfaces.py:finish():202] Joined disk monitor
2024-08-07 18:12:15,174 DEBUG   SystemMonitor:19868 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-07 18:12:15,195 INFO    HandlerThread:19868 [interfaces.py:finish():202] Joined gpu monitor
2024-08-07 18:12:15,195 INFO    HandlerThread:19868 [interfaces.py:finish():202] Joined memory monitor
2024-08-07 18:12:15,195 INFO    HandlerThread:19868 [interfaces.py:finish():202] Joined network monitor
2024-08-07 18:12:15,195 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,195 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-07 18:12:15,195 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 3
2024-08-07 18:12:15,195 DEBUG   SenderThread:19868 [sender.py:send():382] send: stats
2024-08-07 18:12:15,195 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,195 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-07 18:12:15,195 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,195 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-07 18:12:15,195 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 4
2024-08-07 18:12:15,195 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,195 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-07 18:12:15,195 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,195 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-07 18:12:15,195 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 5
2024-08-07 18:12:15,196 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,196 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-07 18:12:15,196 DEBUG   SenderThread:19868 [sender.py:send():382] send: summary
2024-08-07 18:12:15,196 INFO    SenderThread:19868 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:12:15,196 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,196 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-07 18:12:15,196 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 6
2024-08-07 18:12:15,196 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,196 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-07 18:12:15,196 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,196 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-07 18:12:15,197 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: status_report
2024-08-07 18:12:15,207 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8
2024-08-07 18:12:15,207 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '286776', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:15,207 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96A985979ABA7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:15,269 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 7
2024-08-07 18:12:15,269 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:15,269 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-07 18:12:15,269 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:15,269 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-07 18:12:15,955 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/config.yaml
2024-08-07 18:12:15,956 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/wandb-summary.json
2024-08-07 18:12:15,956 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/output.log
2024-08-07 18:12:16,173 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 18:12:16,195 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62
2024-08-07 18:12:16,196 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5216', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:16,196 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96A989F07D7ED</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:16,337 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025
2024-08-07 18:12:16,337 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:16,337 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96A98A78BD814</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:16,338 INFO    wandb-upload_1:19868 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:12:16,525 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8
2024-08-07 18:12:16,525 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '286776', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:16,525 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96A98B0D59819</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:17,019 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 8
2024-08-07 18:12:17,019 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: poll_exit
2024-08-07 18:12:17,019 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:17,019 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-07 18:12:17,020 DEBUG   SenderThread:19868 [sender.py:send_request():409] send_request: defer
2024-08-07 18:12:17,020 INFO    SenderThread:19868 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-07 18:12:17,020 INFO    SenderThread:19868 [job_builder.py:build():298] Attempting to build job artifact
2024-08-07 18:12:17,020 INFO    SenderThread:19868 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-07 18:12:17,034 INFO    SenderThread:19868 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-07 18:12:17,046 INFO    SenderThread:19868 [sender.py:transition_state():617] send defer: 9
2024-08-07 18:12:17,046 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:12:17,046 INFO    HandlerThread:19868 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-07 18:12:17,046 DEBUG   SenderThread:19868 [sender.py:send():382] send: artifact
2024-08-07 18:12:17,173 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 18:12:17,460 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe
2024-08-07 18:12:17,460 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:17,460 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96A98EA7EFE17</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:17,501 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e
2024-08-07 18:12:17,501 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:17,501 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96A98EA1DC765</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:17,956 INFO    Thread-12 :19868 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181212-ag4ny1nq/files/output.log
2024-08-07 18:12:18,622 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe
2024-08-07 18:12:18,622 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:18,622 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96A992FB09F07</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:18,647 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e
2024-08-07 18:12:18,647 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:18,647 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96A99302545C0</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:18,662 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62
2024-08-07 18:12:18,662 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5216', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:18,662 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96A99321D779F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:18,792 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025
2024-08-07 18:12:18,793 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:18,793 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96A9939E4BC12</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:19,046 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8
2024-08-07 18:12:19,046 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '286776', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:19,046 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96A9947565B9F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:20,732 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe
2024-08-07 18:12:20,732 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:20,732 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96A99AD6276AB</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:20,733 INFO    wandb-upload_3:19868 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:12:20,873 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e
2024-08-07 18:12:20,873 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:20,873 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96A99B4D4CCE7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:20,873 INFO    wandb-upload_4:19868 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:12:22,176 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:12:23,292 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025
2024-08-07 18:12:23,292 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:23,292 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96A9A4612334F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:23,686 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8
2024-08-07 18:12:23,687 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '286776', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:23,687 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96A9A5BE74ECB</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:23,697 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62
2024-08-07 18:12:23,698 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5216', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:23,698 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96A9A5E1DF392</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:25,414 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e
2024-08-07 18:12:25,414 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:25,414 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96A9AC35FE367</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:25,451 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe
2024-08-07 18:12:25,451 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:25,451 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96A9AC6ABEB7B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:27,178 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:12:31,558 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025
2024-08-07 18:12:31,558 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:31,558 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96A9C31DC0179</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:32,181 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:12:32,954 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62
2024-08-07 18:12:32,954 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5216', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:32,954 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96A9C85C4CE28</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:33,441 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8
2024-08-07 18:12:33,442 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '286776', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:33,442 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96A9C96069E97</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:34,483 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe
2024-08-07 18:12:34,483 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:34,483 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96A9CE0FE99FD</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:34,631 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e
2024-08-07 18:12:34,631 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:34,632 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96A9CE903F1EB</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:37,183 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:12:42,185 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:12:47,188 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:12:48,719 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=459f7933d891f0a2774d37ba0540589e9fe4d6844a3fa292fde045c192f4d025
2024-08-07 18:12:48,719 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:48,719 ERROR   wandb-upload_1:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AA02F822F04</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:50,703 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=73593078dd2ab7cbe67033d69757a29dd8539ff5a25718e33d152be64226c4fe
2024-08-07 18:12:50,704 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:50,704 ERROR   wandb-upload_3:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AA0A80B9B78</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:51,466 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101217Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=3d6a23d66a124b8181dae88e8e928a361bf38e8bad7d61632f85de38a655c93e
2024-08-07 18:12:51,466 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:51,466 ERROR   wandb-upload_4:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/503/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AA0D28EF6C6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:51,915 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=03a0e2a823c3dbec1c996ceabb294973b7ffeac5ebd9c51992ae31ae415f6c62
2024-08-07 18:12:51,916 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5216', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:51,916 ERROR   wandb-upload_0:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AA0F01475F3</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:12:52,190 DEBUG   HandlerThread:19868 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:12:52,365 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/ag4ny1nq/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101214Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=b911142e5a4368f21b546143d651227fdffad74dd4cce7800cf9d48afaa2eee8
2024-08-07 18:12:52,365 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '286776', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:12:52,365 ERROR   wandb-upload_2:19868 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/ag4ny1nq/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/ag4ny1nq/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AA10280B034</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
